pkg/common/common.go

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
I extracted those functions to a common package because I thought they are independent of the ticket tracker kind. But, I'll move them in the future if is needed

Fixed in commit d496775f074cc8d2c69c217ab63ffdfc711a21a1